schemas/app/inventory/InventorySettings.json

```diff
@@ -12,6 +12,24 @@
       "create": true,
       "section": "Default"
     },
+    {
+      "fieldname": "itemVisibility",
+      "label": "Item Visibility",
+      "fieldtype": "Select",
+      "options": [
+        {
+          "value": "Inventory Items",
+          "label": "Inventory Items"
+        },
+        {
+          "value": "Non-Inventory Items",
+          "label": "Non-Inventory Items"
+        }
+      ],
+      "default": "Non-Inventory Items",
+      "required": true,
+      "section": "Default"
+    },
```

Comment:
This could have been a `Check` field @Gadha2311 . That will be easier in developing too.